packages/lesswrong/lib/collections/users/custom_fields.js

```diff
@@ -731,7 +731,23 @@ addFieldsDict(Users, {
     canRead: ['guests'],
     canUpdate: [Users.owns, 'sunshineRegiment', 'admins'],
     label: "Auto-collapse comments from GPT2"
-  }
+  },
+  
+  // Admin-only options for configuring Recommendations placement, for experimentation
```

Comment:
My guess is that it's better to not make these blackboxes in the schema, but instead document the fields here and then give it a custom graphQL type of `JSON`